astro/src/content/quickstarts/quickstart-springboot-web.mdx

```diff
@@ -56,188 +41,258 @@ The login flow will look like this after FusionAuth is introduced.
+This controller serves both GET and POST requests on the `/make-change` route. Both methods add the claims to the `profile` attribute like `AccountController.java` does. `GetMapping` adds a new model object of `Change` to the view model. `PostMapping` takes the `Change` object as a parameter and uses the `total` to calculate how many nickels and pennies to return. If `total` is not a valid dollar (`BigDecimal`) value, it adds an error to the `Change` model object. The return string `make-change` tells Spring to render the `make-change.html` template.
 
-### Add the model object
+### Add The Model Object
 
-We need to create the model object referenced in the controllers. From the `base directory` create a new directory `model` and in it add a new class `Change.java`
+Now you'll create the model object referenced in the controllers. In the project base directory `src/main/java/io/quickstart/springweb`, create a new directory called `model` and add a new `Change.java` class to it.
 
 <RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-springboot-web/main/complete-application/src/main/java/io/fusionauth/quickstart/springweb/model/Change.java"
             lang="java" />
 
-This model has four fields
+This model has four fields.
 - `error`: The error string if the input was invalid.
 - `total`: The total amount to divide into pennies and nickels.
-- `nickels`: The number of pennies returned from the "make change" operation.
-- `pennies`: The number of pennies returned from the "make change" operation.
+- `nickels`: The number of nickels returned from the `make change` operation.
+- `pennies`: The number of pennies returned from the `make change` operation.
+
+The model also has a `getMessage` method, which formats a message string to render on the template.
 
-It additionally has a method `getMessage` which formats a message string to render on the template.
+### Add View Templates
 
-### Add view templates
+Now you can add three views for Thymeleaf to render on the routes you defined:
 
-Now you need to add some views for Thymeleaf to render on the routes you just defined. Add three files into `src/main/resources/templates`:
+- `Home`: The welcome page an unauthenticated user will see when they visit your application.
+- `Account`: A protected view the user will see after they log in.
+- `Make Change`: A protected view to demonstrate making an authenticated form post to the backend.
 
-`home.html`
+Create the template files in `src/main/resources/templates`.
```

Comment:
Let them know they will need to create the templates folder.

Reply:
I have made adjustments to the instructions. When you download the starter package if you have added the correct dependencies `Thymeleaf` and `Spring Web`, on the Spring Initializr site, the template folder will already be there in your package.